tests/budgeting/rules/test_rules_view_support.py

```diff
@@ -268,7 +268,7 @@ def test_phase_active_project_draft(
 
 
 @pytest.mark.django_db
-def test_post_phase_project_archived(
+def test_post_voting_phase_project_archived(
```

Comment:
@fuzzylogic2000 I realized that we do not consider is_archived anywhere in the rule, so this tests actually is the same as test_voting_phase_active.. same for testing rule view_vote_count, also there is_archived isnt considered... do you think we nee to add it somewhere to the rule or does this test maybe just not make sense?

Reply:
Haaa, don't know. We do consider the archived for things like vote or support, so that you cannot do that when the proposal is archived, right? Not showing sth. anymore when the porposal was archived does seem a bit intransparent? So, I think: let's keep it like that for now and adapt if needed.
Oh, but the test still makes sense, so that we notice when we do change that and only do that on purpose. I also like, that you can see in the test that it is really meant like that and we didn't just forget about it.